### PR TITLE
added auto start script for desktop environments into utils

### PR DESCRIPTION
**Pull request by guruswarupa**
# Pull Request

## Title
added auto start script for desktop environments into utils

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: tabs/utils/auto_start_manager.sh [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            5) echo "Exiting..."; exit 0 ;;
+            *) echo "Invalid choice. Please try again." ;;
+        esac
+
+        echo "Press [Enter] to continue..."
+        read
+    done
+}
+
+main
```
Comment (jeevithakannan2) [inline tabs/utils/auto_start_manager.sh:113]: checkEnv should be called in all scripts as per @nnyyxxxx

**Comment by jeevithakannan2** [comment]
This method does not work with all desktop environments. Add a check for the current desktop environment

File: tabs/utils/auto_start_manager.sh (hunk 1 above)
Comment (nnyyxxxx) [reply tabs/utils/auto_start_manager.sh:113]: @jeevithakannan2 Only if any of the functions in common-script is being called.

Comment (guruswarupa) [reply tabs/utils/auto_start_manager.sh:113]: uk what, stop comparing other's pull requests and use some brain on why i didn't call those 2 functions and import common-script. i didn't use any of those functions. wherein u had to use escalation_tool. u'll see my other scripts have those called as i have used escalation fn.

Comment (guruswarupa) [reply tabs/utils/auto_start_manager.sh:113]: when a reviewer asks you to do something in your pr, whats the necessity to come and type out here to do the same when you do not cross check how my script runs.

**Comment by guruswarupa** [comment]
> This method does not work with all desktop environments. Add a check for the current desktop environment

yes it works on kde plasma, gnome, xfce. doesn't work for window managers

**Comment by guruswarupa** [comment]
> This method does not work with all desktop environments. Add a check for the current desktop environment

could you name the desktop environment in which it doesn't work?

**Comment by jeevithakannan2** [comment]
> > This method does not work with all desktop environments. Add a check for the current desktop environment
> 
> could you name the desktop environment in which it doesn't work?

Enlightment is one of the DEs i have had issues with in the past it does not work $HOME/.config/autostart


And it would be awesome if you add a warning message for users that it will not work WM

**Comment by ChrisTitusTech** [comment]
Missing Standardization with common script, checks for different configurations like is it running a a supported DE, etc. 

Can't merge this.

**Pull request closed without merging**